docs/source/notes/autograd.rst

```diff
@@ -36,6 +36,52 @@ flow statements, that can change the overall shape and size of the graph at
 every iteration. You don't have to encode all possible paths before you
 launch the training - what you run is what you differentiate.
 
+.. _saved-tensors-doc:
+
+Saved tensors
+^^^^^^^^^^^^^
+
+Some operations need intermediary results to be saved during the forward pass
+in order to execute the backward pass. For example, the function
+:math:`x\mapsto x^2` needs to save the input :math:`x` because it is needed to
+compute the gradient.
```

Comment:
Nit: it could technically save `x^2` so I wouldn't say it "needs to save the input". Maybe just "the function saves the input x to compute the gradient"